addons/point_of_sale/static/src/app/screens/scale_screen/scale_screen.js

```diff
@@ -94,4 +92,24 @@ export class ScaleScreen extends Component {
             this.state.tareLoading = false;
         }, 3000);
     }
+    get productPrice() {
+        const product = this.props.product;
+        const priceUnit = product.get_price(this._activePricelist, 1);
```

Comment:
`priceUnit` is the same as `this.pricePerUom`. Maybe you can use the second one to avoid doing the computation multiple times ?

Reply:
I thought I removed it as it was only used once

Reply:
Oh sorry I did not see you were removing it